packages/kit/src/core/create_manifest_data/index.js

```diff
@@ -375,18 +404,21 @@ function get_pattern(segments, add_trailing_slash) {
  * @param {string} dir
  * @param {string} path
  * @param {import('types/internal').Asset[]} files
+ * @param {string[]} excludedPaths Paths relative to dir which should be excluded from files list.
```

Comment:
you'll need to update the doc and then down below where the variable is used

Reply:
Was just pushing this change.

Reply:
Done.